data/lib/compat/compat.lua

```diff
@@ -1350,6 +1350,18 @@ function doSetGameState(state)
 	return Game.setGameState(state)
 end
 
+function Game.inGameState(state)
+	return Game.inState(state)
```

Comment:
```suggestion
	return Game.getState() == state
```

Drop that C++ thingy